### ISSUE TITLE
Accumulated input instance sharing regression in Input.parse_input_event()

### ISSUE THREAD

**Issue by EIREXE** [godotengine/godot#54066]
### Godot version

3.x (b7ded89fe9c327bc3af9ec5922aac78e7141f5bc)

### System information

Arch Linux

### Issue description

Previously, passing a reused instance to parse_input_event even with accumulated input enabled would not cause the event to be shared between emitted events.

### Steps to reproduce

Create two input events with input accumulation enabled:

```
        Input.set_use_accumulated_input(true)
	var a = InputEventAction.new()
	for action in ["test1", "test2"]:
		a.action = action
		Input.parse_input_event(a)
```

Parse them in another node:

```
func _input(event):
	if event is InputEventAction:
		print(event.action)
```

event.action will return test2 in all cases, in an older version it would return test1 and test2 in alternating fashion

### Minimal reproduction project

[InputBug.zip](https://github.com/godotengine/godot/files/7388525/InputBug.zip)

**Comment by EIREXE** [comment]
> I can reproduce the issue with the MRP. The behavior changed between 3.4 beta 3 and 3.4 beta 4, which likely points at #42220. CC @RandomShaper
> 
> I don't know yet if it's an actual bug or a bugfix, I'll let @RandomShaper assess.
> 
> To me both behaviors seem non-intuitive when reading the description of `set_use_accumulated_input`. I tweaked the project to count how many events are sent and received in 3 seconds, and it's the same number for both. So I'm not sure what accumulation happens at all here.
> 
> [InputBug.zip](https://github.com/godotengine/godot/files/7388742/InputBug.zip)

I think accumulation is done in the OS implementation, and since we are firing events ourselves it's likely those don't get accumulated, but don't quote me on that.

**Comment by akien-mga** [comment]
So given the above discussion, I think this just requires some better documentation?

**Comment by Rubonnek** [comment]
This issue has been fixed in `master` at #64423 and in `3.x` at #64424.

Running the MRP now yields the following message:

```
WARNING: An input event object is being parsed more than once in the same frame, which is unsafe.
If you are generating events in a script, you have to instantiate a new event instead of sending the same one more than once, unless the original one was sent on an earlier frame.
You can call duplicate() on the event to get a new instance with identical values.
```

Closing.
